griptape/drivers/sql/amazon_redshift_sql_driver.py

```diff
@@ -0,0 +1,112 @@
+import time
+from typing import Optional
+import boto3
+from griptape.drivers import BaseSqlDriver
+from attr import Factory, define, field
+
+
+@define
+class AmazonRedshiftSqlDriver(BaseSqlDriver):
+    database: str = field(kw_only=True)
+    session: boto3.session = field(kw_only=True)
+    cluster_identifier: Optional[str] = field(default=None, kw_only=True)
+    workgroup_name: Optional[str] = field(default=None, kw_only=True)
+    db_user: Optional[str] = field(default=None, kw_only=True)
```

Comment:
Should this also be required? Or are there cases where it can be `None`?

Reply:
The db_user is optional and can be None. If it is None, then the [database user name is derived from the IAM identity](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/redshift-data/client/execute_statement.html).

Reply:
Gotcha